benchmark/build.gradle.kts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "com.wire.benchmark"
+    compileSdk = 34
```

Comment:
Can't we use the same versions already used by `app`?

Now we'd need to change these values twice.

Maybe create a convention plugin to live together with `AndroidApplicationConventionPlugin` and share the same values.

Reply:
Not possible API 34 is required to run the benchmark and our app uses 33.

I think we can keep it like this for now, it's a separate module that's not involved in the app or the CI. And later we when we upgrade the API we can use `AndroidLibraryConventionPlugin`

Reply:
Can't we bump to 34?